koza/io/utils.py

```diff
@@ -32,7 +32,7 @@ def open_resource(resource: Union[str, PathLike], compression: CompressionType =
     :return: str, next line in resource
 
     """
-    if isinstance(resource, PosixPath) and Path(resource).exists():
```

Comment:
I wish I remembered why I wrote this

Reply:
I did a git blame, thinking "I wonder why Kent wrote that" and found my name on it, but I didn't keep chasing it back further to see if the PosixPath check was there before that.